### PR TITLE
fix: update BoxKnownProps to reflect support for width props

### PR DESCRIPTION
**Pull request by philipwindeyer**
## Description

This corrects the "BoxProps" type def for the `<Box />` component, aligning it with the components supported props, namely `width`.

💁‍

## Related issues

Came up when I was playing with button widths in this card: https://app.shortcut.com/qantasaccommodation/story/77586/modal-update-payment-details-modal-over-the-booking-details

💪

## Looks like this

NA

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [x] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan) [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [x] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

**Comment by ehyland** [comment]
👷 Deploy Preview for *roo-ui* processing.


🔨 Explore the source changes: 68c4c7b1015d5c4b40795f38fbcb6e583429f259

🔍 Inspect the deploy log: [https://app.netlify.com/sites/roo-ui/deploys/6153ae0fc777760007ae0f19](https://app.netlify.com/sites/roo-ui/deploys/6153ae0fc777760007ae0f19?utm_source=github&utm_campaign=bot_dl)

**Review by mrm007 — APPROVED** [review]

File: src/components/index.d.ts [hunk 1]
```diff
@@ -91,8 +91,6 @@ declare module 'roo-ui/components' {
       SS.BottomProps,
       SS.LeftProps,
       SS.OverflowProps,
-      SS.WidthProps,
-      SS.HeightProps,
       SS.FontSizeProps,
```
Comment (philipwindeyer) [inline src/components/index.d.ts:94]: NB: This is the `BoxKnownProps` def. It was extending WidthProps and HeightProps twice unnecessarily

**Review by Maxmus1989 — APPROVED** [review]
✈️

**Pull request merged**